src/d/llvm/expression.d

```diff
@@ -709,6 +709,24 @@ struct ExpressionGen {
 			case SPad:
 				return LLVMBuildSExt(builder, value, type, "");
 
+			case FloatTrunc:
+				return LLVMBuildFPTrunc(builder, value, type, "");
+
+			case FloatExtend:
+				return LLVMBuildFPExt(builder, value, type, "");
+
+			case FloatToSigned:
+				return LLVMBuildFPToSI(builder, value, type, "");
+
+			case FloatToUnsigned:
+				return LLVMBuildFPToUI(builder, value, type, "");
+
+			case SignedToFloat:
+				return LLVMBuildSIToFP(builder, value, type, "");
+
+			case UnsignedToFloat:
+				return LLVMBuildUIToFP(builder, value, type, "");
+
```

Comment:
This definitively shouldn't come in the middle of integer pad/trunc.

Reply:
no longer the case